test/IISIntegration.FunctionalTests/Utilities/IISAncmValidator.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Security.Principal;
+using System.Xml.Linq;
+
+namespace Microsoft.AspNetCore.Server.IntegrationTesting
+{
+
+    // Adapted from IISExpressAncmSchema
+    public class IISAncmValidator
+    {
+        public static bool IsMet { get; }
+        public static string SkipReason { get; }
+
+        static IISAncmValidator()
+        {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                SkipReason = "IIS tests can only be run on Windows";
+                return;
+            }
+
+            var identity = WindowsIdentity.GetCurrent();
+            var principal = new WindowsPrincipal(identity);
+            if (!principal.IsInRole(WindowsBuiltInRole.Administrator))
```

Comment:
I didn't see an answer to the elevation question. Do the tests pass without elevation?